Doc/release-notes/master.txt

```diff
@@ -9,3 +9,4 @@ Updates since v1.5.0
 --------------------
 
 * Update Eigen from 3.3.7 to 3.3.9
+* Added option for GDAL 2.4.4
```

Comment:
If GDAL 2.4.4 is now the default (which is fine by me) then this comment should indicate that 2.4.4 is now the default, not just an option.